### DISCUSSION TITLE
Use string array in a localParams

### DISCUSSION THREAD

**Discussion by sodoto** [microsoft/RulesEngine#345] [Q&A]
Hi, thank you for your work. 

What would be the best way to use an array or a list in the localParams.
Ex.: If I have a string array
`var ips = new[]{"192.168.1.2", "192.168.1.3", "10.10.1.58", };`

And I want to use it in a localParams. I tried this but it didn't worked.
`new ScopedParam{ Name = "ips", Expression = $"new []({string.Join(",", ips)})"}`

Is this possible or it would be better to use the array in an input?

Thank you!

**Comment by abbasc52** [comment]
I have picked this from a working sample -
``` json
"LocalParams": [
				{
					"Name": "excludedTitles",
					"Expression": "new string[]{\"Architect\",\"Engineer\"}.Select(it.ToLower())"
				}
			],
```
If you specify the type along with new, it should work
